src/main/scala/io/iohk/ethereum/blockchain/sync/regular/RegularSync.scala

```diff
@@ -78,22 +96,49 @@ class RegularSync(
       "block-importer"
     )
 
+  implicit val system: ActorSystem = context.system
+  implicit val ec: ExecutionContext = context.dispatcher
+
   val printFetcherSchedule: Cancellable =
     scheduler.scheduleWithFixedDelay(
       syncConfig.printStatusInterval,
       syncConfig.printStatusInterval,
       fetcher.toClassic,
       BlockFetcher.PrintStatus
-    )(context.dispatcher)
+    )
 
   override def receive: Receive = running(
     ProgressState(startedFetching = false, initialBlock = 0, currentBlock = 0, bestKnownNetworkBlock = 0)
   )
 
+  private def startNewFlow() =
+    PeerEventBusActor
+      .messageSource(
+        peerEventBus,
+        PeerEventBusActor.SubscriptionClassifier
+          .MessageClassifier(
+            Set(Codes.BlockBodiesCode, Codes.BlockHeadersCode),
+            PeerEventBusActor.PeerSelector.AllPeers
+          )
+      )
+      .buffer(256, OverflowStrategy.fail)
```

Comment:
I guess in the future this should be configurable, in case we need to tweak it?

Reply:
Well that depends a bit on what the full logic will look like later, since incoming data depends on requests. So if processing lags a bit (for whatever data), fewer requests for more data will be done. This is something we'll be able to reason about.